app/helpers/insured/families_helper.rb

```diff
@@ -241,7 +241,7 @@ def build_link_for_sep_type(sep, link_title = nil, family_id = nil)
       # Use turbolinks: false, to avoid calling controller action twice.
       # TODO: Refactor Shop For Planss as a translation at some point
       link_path = family_id.present? ? insured_family_members_path(sep_id: sep.id, qle_id: qle.id, family_id: family_id) : insured_family_members_path(sep_id: sep.id, qle_id: qle.id)
-      link_to link_title.presence || 'Shop for Plans', link_path, class: 'btn btn-default', data: {turbolinks: false}
+      link_to link_title.presence || 'Shop for Plans', link_path, data: {turbolinks: false}
```

Comment:
This looks like a good opportunity to refactor a hardcoded english string that already exists in a translation file.  I think we can call `l10n("insured.shop_for_plans")` in place of 'Shop for Plans'.